src/components/common/Navbar/navbar.scss

```diff
@@ -169,6 +169,22 @@
 }
 
 .navlinks__list {
+  [data-postal-code-bar] {
+    display: none;
+
+    @include media(">=notebook") {
+      display: block;
+    }
+
+    #postal-code-bar-edit, #postal-code-bar-caret {
```

Comment:
I did it because the edit text and the caret are not supposed to appear on the Desktop.

Reply:
We have some existing CSS helper classes that help show only for mobile or only for desktop, they're `.display-mobile` and `.hidden-mobile`. I think we can use it instead and possibly remove all the added CSS to this file. Shall we try it? I can help you.

Reply:
We might need to change the `Icon` component to allow it to receive CSS classes so we can pass in `.display-mobile`, etc.